.github/workflows/CreatePullRequest.yml

```diff
@@ -0,0 +1,38 @@
+name: Create Pull Request
+
+on:
+  pull_request:
+    types: ["opened"]
+
+jobs:
+  assign_and_create_card:
+    name: Assign issue to sender and create Kanban card
+    runs-on: ubuntu-latest
+    # PRs from forks don't have required token authorization
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+    steps:
+      # https://github.com/actions/github-script
+      - uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const IN_PROGRESS_COLUMN = 4971952;
+            //
+            async function addAssignee(issue, login) {
+                console.log("Assigning to: " + login);
+                await github.issues.addAssignees({
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    issue_number: issue.number,
+                    assignees: [login]
+                });
+            }
+            //
+            if(context.payload.pull_request.body.match(/Fixes\s*#\d+/gi)) {
```

Comment:
could you please take into consideration all the accepted keywords? https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

even if our convention is "fixes", the github action should take all into account